pkg/agent/controller/egress/egress_controller.go

```diff
@@ -155,6 +172,17 @@ func NewEgressController(
 		localIPDetector:      localIPDetector,
 		idAllocator:          newIDAllocator(minEgressMark, maxEgressMark),
 	}
+	ipAssigner, err := NewIPAssigner(nodeIP, defaultEgressRunDir)
+	if err != nil {
+		return nil, fmt.Errorf("initializing egressIP assigner failed: %v", err)
+	}
+	c.ipAssigner = ipAssigner
+
+	cluster, err := memberlist.NewCluster(clusterPort, nodeIP, nodeName, nodeInformer, externalIPPoolInformer)
+	if err != nil {
+		return nil, fmt.Errorf("initializing Egress node memberlist cluster failed: %v", err)
```

Comment:
nit: it should be "Node" not "node"

but I am not sure why we include "node" in the error message in the first place

Reply:
done